src/Compilers/VisualBasic/Test/Emit/PDB/PDBAsyncTests.vb

```diff
@@ -809,95 +809,91 @@ End Module
     IL_001b:  ldarg.0
     IL_001c:  ldfld      ""M.VB$StateMachine_0_F.$VB$ResumableLocal_$VB$Closure_$0 As M._Closure$__0-0""
     IL_0021:  stfld      ""M.VB$StateMachine_0_F.$U1 As M._Closure$__0-0""
-    IL_0026:  ldarg.0
-    IL_0027:  ldfld      ""M.VB$StateMachine_0_F.$U1 As M._Closure$__0-0""
-    IL_002c:  ldfld      ""M._Closure$__0-0.$VB$Local_z As Integer""
```

Comment:
This is the only actual difference. We would load z and then pop it to trigger side effects (irrelevant here since the Closure can't be null, but incorrectly since the NRE only occurs on assignment)